StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionCellView.swift

```diff
@@ -72,8 +73,9 @@ final class InstitutionCellView: UIView {
         return activityIndicator
     }()
 
-    override init(frame: CGRect) {
-        super.init(frame: frame)
+    init(theme: FinancialConnectionsTheme?) {
```

Comment:
is that weird that the theme is NULL here but `InstitutionCellView` is used in a place where it isn't expected to be? (...because its inside of Auth Flow where the the theme is now non-null for "after synchronize")? https://github.com/stripe/stripe-ios/pull/3793/files

Reply:
this is in a bunch of places